var/spack/repos/builtin/packages/brayns/package.py

```diff
@@ -36,7 +36,8 @@ class Brayns(CMakePackage):
     depends_on('ispc', type='build')
     depends_on('ninja', type='build')
 
-    depends_on('assimp', when='+assimp')
+    depends_on('assimp@5.0.1', when='+assimp')
```

Comment:
Maybe this?
```suggestion
    depends_on('assimp@5.0.1', when='@1.1.1: +assimp')
```

I myself am never sure how all these constraints will work when you leave one generic :( I have the gut feeling that if you did not constrain `assimp`, it would work.

Reply:
Do you mean to leave it to pick only the latest version?